Add interactive schema viewer references to migration documentation (#12558)
opensearch-project/documentation-website: _migration-assistant/playbook-solr-8.11-to-opensearch-3.md

```
{% include copy.html %}

> Always verify field names against `workflow configure sample` for your installed version, or browse the [Migration Assistant Schema Viewer](https://opensearch-project.github.io/opensearch-migrations/) for a complete interactive field reference.

Reviewer: ```suggestion
Always verify field names against `workflow configure sample` for your installed version, or browse the [Migration Assistant Schema Viewer](https://opensearch-project.github.io/opensearch-migrations/) for an interactive field reference.
```
